testes/matelcli/sync.hs

```diff
@@ -0,0 +1,12 @@
+-- | This test suite determines whether or not @matelcli@ can
+-- successfully @sync@.
+module Main where
+import System.Exit;
+import System.Process;
+
+-- | @main@ attempts to use @matelcli@ to use the Matrix API's "@sync@"
+-- command, returning the exit code which is output by @matelcli@.
+main :: IO ExitCode;
+main = (\(a,_,_) -> a) <$> readProcessWithExitCode "cabal" [] []
```

Comment:
This thing is super terribad.  Because `argh` is unused, this thing just runs `cabal`, which should always exit with a status code of 0.

What idiot decides to merge this request?

Aw, dang.